### DISCUSSION TITLE
Why fork?

### DISCUSSION THREAD

**Discussion by FooBarWidget** [paritytech/cachepot#154] [General]
Why is cachepot a fork? Isn't security a core concern for sccache too? Or is this fork mostly about the ability to develop independently without the sccache maintainers' approval?

**Comment by drahnr** [comment]
There was a time when Mozilla layoffs conincided with a a cache PR review dormancy and general radio silence. There is an ongoing conversation regarding direction and alignment between the projects.

I think the threat model needs to be clearly defined for cachepotl, but that's a bit out of scope for this PR.

**Comment by Xanewok** [comment]
I think it makes sense to move this topic to Discussions so I went ahead and converted it. I hope that's okay with y'all!
